main.go

```diff
@@ -124,6 +117,38 @@ func main() {
 	}
 }
 
+// Function to go to GetHub and process the passed pull requests
+func processPullRequests(ids []int, client *github.Client, config Config, repo string) []*github.PullRequest {
+
+	jobs := make(chan *github.PullRequest)
+
+	// List of Pull Requests
+	pulls := []*github.PullRequest{}
+
+	for _, item := range ids {
+		go func(client *github.Client, org string, r string, id int) {
+			pr, _, err := client.PullRequests.Get(org, r, id)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			jobs <- pr
+
+		}(client, config.Github.Org, repo, item)
+	}
+
+	for {
```

Comment:
instead of the infinite loop, you can [specify to loop as many times as you have jobs](https://golang.org/doc/effective_go.html#parallel). Alternatively, you can use a sync.WaitGroup and a buffered channel to collect the responses.